bitsandbytes/nn/modules.py

```diff
@@ -492,8 +521,9 @@ def forward(self, x: torch.Tensor):
             x = x.to(self.compute_dtype)
 
         bias = None if self.bias is None else self.bias.to(self.compute_dtype)
+        weight = self.weight.t() if len(self.weight.shape) == 2 else self.weight
```

Comment:
It's not clear to me when `len(self.weight.shape) != 2`.

Reply:
already in the comment